### PR TITLE
Add footer aa1 to implement aa1

### PR DESCRIPTION
**Pull request by pinterid**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no footer.

**What is the new behavior (if this is a feature change)?**
- Now there is a footer.

**Other information**:
- Ref: 🐍

### CODE REVIEW

**Review by schettn — APPROVED** [review]
Looks good

File: templates/base.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
```
Comment (kleberbaum) [inline templates/base.html]: Please remove this newline.
```suggestion
```

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by Aichnerc — APPROVED** [review]
Lgtm

**Comment by pinterid** [comment]
Codacy was ignored on this PR because it was not able to analyze it.

**Pull request merged**